IdentityCore/src/cache/MSIDSharedCacheAccessor.h

```diff
@@ -50,6 +50,10 @@
                             context:(id<MSIDRequestContext>)context
                               error:(NSError **)error;
 
+- (NSArray<MSIDBaseToken *> *)allTokensForAccount:(MSIDAccount *)account
```

Comment:
just curious, where is this necessary?

Reply:
we use it in msal, see here:https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/266/files#diff-10a74bc3b5085f9e89a569f246fe68efR580

Reply:
I believe it's better then to add an API removeTokensForAccount: so we avoid querying all items and then doing a separate removal call for each item.

Reply:
we can do that, but we still use "allTokensForAccount" in test app, on cache screen.